release/README.md

```diff
@@ -30,7 +30,7 @@
   cd release
   env -i HOME="$HOME" PATH="$PATH" SSH_AUTH_SOCK="$SSH_AUTH_SOCK" RELEASE_TAG="$RELEASE_TAG" USER="$USER" PRERELEASE="$PRERELEASE" ./release.sh
   ```
-* Make sure `main` branch is properly merged to `docs-0.7`:
+* Wait for A post release Github job to merge `main` branch to `docs-0.7` branch; you can watch for it here:
```

Comment:
I would be a bit more explicit about the job we're waiting for, something along the lines of:

```suggestion
* Wait for the [name of workflow](link to workflow.yml] to complete; this workflow automatically merges `main` into `docs-0.7`. You can watch for it here:
```

Reply:
Additionally it could be helpful to include a description about what to do if the automatic merge fails, which will occur from time to time.